docs/management/advanced-options.asciidoc

```diff
@@ -204,6 +204,16 @@ be inconsistent because different shards might be in different refresh states.
 Searching through frozen indices
 might increase the search time. This setting is off by default. Users must opt-in to include frozen indices.
 
+[float]
+[[kibana-siem-settings]]
+=== SIEM settings
+
+[horizontal]
+`siem:defaultAnomalyScore`:: The threshold above which anomalies are displayed in the SIEM app.
+`siem:defaultIndex`:: A comma-delimited list of index patterns from which the SIEM app collects events. 
```

Comment:
Should the description say "index pattern" or "index" . The text in the UI is "Default index"

Reply:
Yes, thanks Gail - I corrected the text.

@spong - can you please also review to check all the descriptions are technically correct - thanks.